lib/src/instance/extension_instance.dart

```diff
@@ -10,13 +11,36 @@ extension Inst on GetInterface {
           {String tag, bool permanent = false}) async =>
       GetInstance().putAsync<S>(builder, tag: tag, permanent: permanent);
 
+  /// Creates a new Instance<[S]> from the <[S]>[builder] callback.
+  /// Every time [find]<[S]>() is used, it calls the builder method to generate
+  /// a new Instance [S].
+  ///
+  /// Example:
+  ///
+  /// ```create(() => Repl());
+  /// Repl a = find();
+  /// Repl b = find();
+  /// print(a==b); (false)```
```

Comment:
```suggestion
  /// ```
  /// create(() => Repl());
  /// Repl a = find();
  /// Repl b = find();
  /// print(a==b); (false)
  ///```
```

just a small suggestion, i think is better like that
the rest of the code i don't have the knowledge to review 😅